mappings/net/minecraft/client/texture/TexturePlayerSkin.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS none/bwu net/minecraft/client/texture/TexturePlayerSkin
+	CLASS none/bwu$1
+		METHOD run run ()V
+	FIELD g LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD h DOWNLOAD_THREAD_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
```

Comment:
If it's mutable then it shouldn't be written in `ALL_CAPS` in my opinion (exceptions being things set on game initialization)..?